containers/Dockerfile.template

```diff
@@ -50,7 +50,8 @@ RUN if [ -n "$PR_NUMBER" ]; then git fetch origin pull/${PR_NUMBER}/head:pr-${PR
 # if it is a TAG trigger checkout the tag
 RUN if [ -n "$TAG" ]; then git checkout "$TAG";fi
 
-RUN python3.9 -m ensurepip
+RUN python3.9 -m ensurepip --upgrade --default-pip
+RUN python3.9 -m pip install --upgrade pip setuptools=70.0.0
```

Comment:
@tsebastiani docker image CI is failing when running this step: https://github.com/krkn-chaos/krkn/actions/runs/14032400212/job/39282751658?pr=781